build/build-sdk-images/cpp/Dockerfile

```diff
@@ -27,5 +33,9 @@ RUN cmake --version
 WORKDIR /go/src/agones.dev/agones
 
 # code generation scripts
-COPY *.sh /root/
-RUN chmod +x /root/*.sh
+COPY *.sh /home/agones-dev/
+RUN chmod +x /home/agones-dev/*.sh
+
+RUN chown --silent --no-dereference --recursive ${USER_ID:-1337}:${GROUP_ID:-1338} /home/agones-dev /go
```

Comment:
may need to allow "other" to write in these dirs? Root should have privileges though, right?